app/views/org_admin/questions/_show.html.erb

```diff
@@ -58,6 +58,36 @@
           <% end %>
         </dl>
       </div>
+      <div class="col-md-4">
+        <% if guidance.present? %>
```

Comment:
This is referencing the "annotation" guidance associated with the question. Themed guidance should be independent of that.